Modules/Loadable/SubjectHierarchy/Widgets/qMRMLSubjectHierarchyTreeView.cxx

```diff
@@ -1298,6 +1301,63 @@ bool qMRMLSubjectHierarchyTreeView::multiSelection()
 }
 
 //-----------------------------------------------------------------------------
+vtkIdType qMRMLSubjectHierarchyTreeView::firstSelectedInHierarchyFrom(vtkIdType itemID)
```

Comment:
I think the method name is not descriptive enough, it doesn't specify first of what is looked for. Also, the convention in SH is that we call "sub-hierarchies" branches. I'd suggest calling this firstSelectedSubjectHierarchyItemInBranch

Reply:
great name :+1: I was not happy about the current one.